src/components/updateUserProfile/useImagePreview.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect, useCallback } from "react";
+import { UseFormReturn } from "react-hook-form";
+import {
+  handleFilePreview,
+  handleImageUpload,
+} from "../../utils/imageuploadUtil";
+import { TuserSchema } from "../../utils/userSchema";
+import { toast } from "react-toastify";
+
+const DEFAULT_IMAGE =
+  "https://t4.ftcdn.net/jpg/01/24/65/69/360_F_124656969_x3y8YVzvrqFZyv3YLWNo6PJaC88SYxqM.jpg";
```

Comment:
-Let use our local images only
- Avoid outside images links

Reply:
I have changed that